internal/parameterizer/parameterizer_test.go

```diff
@@ -0,0 +1,247 @@
+/*
+Copyright IBM Corporation 2020
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package parameterize_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/google/go-cmp/cmp/cmpopts"
+
+	"github.com/google/go-cmp/cmp"
+
+	common "github.com/konveyor/move2kube/internal/common"
+	parameterize "github.com/konveyor/move2kube/internal/parameterizer"
+	"github.com/konveyor/move2kube/internal/types"
+	irtypes "github.com/konveyor/move2kube/internal/types"
+	plantypes "github.com/konveyor/move2kube/types/plan"
+	log "github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func TestParameterizer(t *testing.T) {
+	log.SetLevel(log.DebugLevel)
+
+	t.Run("1.IR with no services, no storage", func(t *testing.T) {
+
+		ir := getIRWithoutServices()
+		irAfter, err := parameterize.Parameterize(ir)
+
+		if err != nil {
+			t.Fatal("Failed to parameterize. Error:", err)
+		}
+
+		irAfter.TargetClusterSpec.Host = ""
+
+		if !cmp.Equal(ir, irAfter, cmpopts.EquateEmpty()) {
+			t.Fatalf("TODO explanation. Difference:\n%s", cmp.Diff(ir, irAfter))
+		}
+
+	})
+
+	/*
+		t.Run("2.IR containing services that have no containers", func(t *testing.T) {
+
+			ir := getIRWithServicesAndWithoutContainers()
+			irAfter, err := parameterize.Parameterize(ir)
+			if err != nil {
+				t.Fatal("Failed to parameterize. Error:", err)
+			}
+
+			// oracle
+			// if the service has no containers, then no changes will be done
+			// ir = irAfter
+			if !cmp.Equal(ir, irAfter, cmpopts.EquateEmpty()) {
+				t.Fatalf("TODO explanation. Difference:\n%s", cmp.Diff(ir, irAfter))
+			}
+
+		})
+	*/
+
+	t.Run("3.IR containing services with containers", func(t *testing.T) {
+
+		ir := getIRWithServicesAndContainers()
+		irAfter, err := parameterize.Parameterize(ir)
+		if err != nil {
+			t.Fatal("Failed to parameterize. Error:", err)
+		}
+		fmt.Printf("%T", irAfter)
```

Comment:
Don't using `fmt.Print` use the logging package `log`
You have already imported it.
This should probably be `log.Infof` or `log.Debugf`
Also why are we printing the type? `%T` We already know what type it is `github.com/konveyor/move2kube/internal/types.IR`